### PR TITLE
User menu

### PR DESCRIPTION
**Pull request by Timuriagudin**
Add user.menu.js, userMenu.page.js, userMenu.page.expected.js

### CODE REVIEW

File: test/specs/regression/testBasePage.js [hunk 1]
```diff
@@ -1,10 +1,10 @@
-import LoginPage from '../../../pageobjects/login.page';
```
Comment (Anastasiia-Bandurova) [inline test/specs/regression/testBasePage.js:1]: why did you add commented code to PR?

File: test/specs/smoke/testBasePage.js [hunk 2]
```diff
@@ -1,13 +1,13 @@
-import LoginPage from '../../../pageobjects/login.page';
```
Comment (Anastasiia-Bandurova) [inline test/specs/smoke/testBasePage.js:1]: again commented code

**Review by Anastasiia-Bandurova — APPROVED** [review]

**Review by dannywhy — APPROVED** [review]
